### PR TITLE
Add default value for the arguments to simplify the commands in tutorial.

### PR DESCRIPTION
**Pull request by brightcoder01**

### CODE REVIEW

**Review by workingloong — DISMISSED** [review]
Excellent!

File: elasticdl_client/common/args.py [hunk 1]
```diff
@@ -209,7 +209,7 @@ def add_common_params(parser):
         "default to master_resource_request",
     )
     parser.add_argument(
-        "--num_workers", type=int, help="Number of workers", default=0
+        "--num_workers", type=int, help="Number of workers", default=1
```
Comment (skydoorkai) [inline elasticdl_client/common/args.py]: default=2 to make it distributed?

File: elasticdl_client/common/args.py [hunk 2]
```diff
@@ -309,7 +309,7 @@ def add_common_params(parser):
         "--num_minibatches_per_task",
         type=int,
         help="The number of minibatches per task",
-        required=True,
+        default=2,
```
Comment (skydoorkai) [inline elasticdl_client/common/args.py]: Is `2` too small for perf? 8?

File: elasticdl_client/common/args.py (hunk 1 above)
Comment (brightcoder01) [reply elasticdl_client/common/args.py]: Good idea.

File: elasticdl_client/common/args.py (hunk 2 above)
Comment (brightcoder01) [reply elasticdl_client/common/args.py]: Good idea.

**Review by skydoorkai — APPROVED** [review]

**Pull request merged**